src/components/TextWithSelectField/TextWithSelectField.tsx

```diff
@@ -100,19 +88,12 @@ const TextWithSelectField: React.FC<TextWithSelectFieldProps> = ({
               <Spinner />
             </Box>
           ) : (
-            <SingleSelectField
+            <Select
               name={selectFieldName}
-              onChange={handleSelectChange}
+              onChange={value => change({ target: { name: selectFieldName, value } })}
               value={selectFieldValue}
-              className={selectFieldClassName}
-              InputProps={{
-                classes: {
-                  input: classes.noBackground,
-                  root: classes.input,
-                  notchedOutline: classes.noBorder,
-                },
-              }}
-              choices={choices}
+              className={clsx("noBorder", selectFieldClassName)}
```

Comment:
can't we use sprinkles here?

Reply:
Nope, I've tried but I couldn't be able to override border, because class created by sprinkles was override by component styles